README.md

```diff
@@ -908,6 +908,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Free Professional Resume Templates & Editor](https://www.overleaf.com/latex/templates/tagged/cv) - Free platform with lots of Resume templates of Experienced Professionals, ready to clone and edit fully and download, ATS optimized.
   * [DeepLearning.AI Short Courses](https://www.deeplearning.ai/short-courses/) - Free short courses from industry-leading experts to get hands-on experience with the latest generative AI tools and techniques in an hour or less.
   * [Roadmap.sh](https://roadmap.sh) - Free learning roadmaps covering all aspects of development from Blockchain to UX Design.
+  * [Dubai Software Engineer Salary and Cost of Living Calculator]([https://ayehigh.com](https://zerotaxjobs.com/tools/dubai-software-engineer-salary-calculator)) - Free tool to generate a financial report for Software Engineer earnings and living expenses in Dubai, United Arab Emirates.
```

Comment:
Thanks. Sorry but I think this is too narrow focussed for this list being a generic list with global applicability.